### ISSUE TITLE
List enum variants in "Navigate to"

### ISSUE THREAD

**Issue by dvtomas** [intellij-rust/intellij-rust#2389]
I wish *Navigate to class (Ctrl-N)* or *Navigate to symbol (Ctrl-Alt-Shift-N)* would also offer enum variants. Those can be complex types, often I can easily guess name of a specific variant I'm interested in, but I'm not sure about the enum name, so I don't know what to type into the "Navigate" filter to reach the variant.

Typical example from my codebase that has just made me submit this issue :-)

```rust
pub enum CoreStateSpec {
    HWStarting,
    HWUp { boards: Vec<BoardSN> },
    MeasurementStarting { spec: StartMeasurementSpec },
    Measuring {
        data_processor: DataProcessor,
        calibration_processor: CalibrationProcessor,
        measurement_data: MeasurementData,
        savers_trigger: PrePostTrigger<usize>,
        savers_muted: bool,
        savers: Savers<FS3DataVector>,
    },
}
```

I know I want to navigate to `MeasurementStarting`, but I don't remember that the enum's name is `CoreStateSpec`

I'm not sure if it would be better to have them listed in *Navigate to class* or *Navigate to symbol*, perhaps the former makes more sense but I don't really care. I just want to be able to easily navigate to them.

**Comment by jwmatthews** [comment]
Interested to see the same support added.

**Comment by dvtomas** [comment]
It seems it is already implemented as Go to symbol (Ctrl-Shift-N), at least with Intellij Idea 2022.1.1

**Comment by jwmatthews** [comment]
Thank you @dvtomas, I confirmed I'm able to navigate from an enum to where it was declared now.
Error is on my side, I'm new to Rust and using Intellij-rust

Below is what happened for anyone else who stumbles on this in future.

I had a small project with 2 source files:  `src/lib.rs` and `src/main.rs`
I was able to build/run from the terminal with cargo with no issues, yet I didn't realize the IDE was warning me that I had not attached the lib module.   

IDE was displaying: "File not included in module tree, analysis is not available" when I opened `src/main.rs`
I clicked the 'Attach file to lib.rs` and then I was able to navigate the enum as expected.
